python/pipeline/utils/galvo_corrections.py

```diff
@@ -0,0 +1,61 @@
+from scipy.interpolate import interp1d, interp2d
+import numpy as np
+
+
+def correct_motion(img, xymotion):
```

Comment:
For this function (`correct_motion`) and `correct_raster`, it would be really useful to have some doc string explaining what these two functions do. Please add some simple documentation.